### ISSUE TITLE
On Firefox 75, the "megabar" is pushed downward

### ISSUE THREAD

**Issue by alfonsojon** [Zonnev/elementaryos-firefox-theme#65]
Firefox 75 has a new awesomebar called the "megabar", which has a rather large URL bar that auto-expands. I've disabled this feature as I find it annoying; however, Mozilla will be preventing users from doing that. The CSS needs to be updated for Firefox 75+ to accomodate the megabar, as the box gets pushed off into the tab bar/bookmarks bar.

**Comment by Zonnev** [comment]
Hello, I don't find the need to update userChrome for the megabar, as it behaves normal. The megabar box doesn't push things away, it goes over tab and bookmarks bar which is normal behaviour. I understand that tab bar isn't visible when megabar is active and that's probably why you disabled the megabar. It just takes a bit of time getting used to working with the megabar. What I do is clicking on an empty field to get rid of megabar and then look for the tab that I want to see. This way the megabar won't come out covering all tabs making them invisible.

**Comment by Zonnev** [comment]
I adjusted userChrome a bit so tab titles are more visible. Be sure to use the latest userChrome.

**Comment by Vento09** [comment]
I was facing the same issue. I wasn't liking the megabar over my tabbar like the image below

![Captura de tela de 2020-04-17 15-18-53](https://user-images.githubusercontent.com/30022019/79614105-5af89100-80ef-11ea-862a-bdd57524b514.png)

And I wasn't liking the height of the alternative search below the bar.

![Captura de tela de 2020-04-17 16-06-38](https://user-images.githubusercontent.com/30022019/79615400-3651e880-80f2-11ea-82ea-1cc5ea625866.png)

So I started searching and put some codes in the userchrome and stayed better, at least the megabar isn't over my tabbar.

![Captura de tela de 2020-04-17 20-55-14](https://user-images.githubusercontent.com/30022019/79614593-5bddf280-80f0-11ea-902d-a4759b12d41c.png)

And I have decreased the height of alternative search option. 

![Captura de tela de 2020-04-17 18-47-17](https://user-images.githubusercontent.com/30022019/79614604-626c6a00-80f0-11ea-8424-ea0a51ec0a25.png)

The code is down there. I wait it helps you both : ).

>#urlbar[breakout][breakout-extend]{
>    left: -7.5px !important; /\* take the bar a little bit to right\*/
>    width:102%!important; /\* expand the bar\*/
>    padding: 3px !important;
>}
>
> #urlbar[breakout][breakout-extend] #urlbar-input-container{
>    height: 35px !important;
>    padding: 5px !important;
>} 
>
>.urlbarView-row{
>   padding: 3px !important;
>}
>
>/\*decrease the search one off engine\*/
>.search-one-offs{
>    padding-top: 6px !important;
>    height: 40px !important; 
>}
